clients/fides-js/src/components/fides.css

```diff
@@ -53,18 +53,18 @@
 
   /* Everything else */
   --fides-overlay-font-family: Inter, sans-serif;
-  --fides-overlay-font-size-body-small: 0.875em;
-  --fides-overlay-font-size-body: 0.95em;
+  --fides-overlay-font-size-body-small: 0.75em;
+  --fides-overlay-font-size-body: 0.875em;
```

Comment:
wow do we really want it this small? it's getting hard to read, unless I have something misconfigured
![image](https://github.com/ethyca/fides/assets/24641006/ac9d2b0a-4c92-4e04-b728-35c7d498713b)

Reply:
I think this needs to use `rem`s instead. will fix